tools/generator-bot-template-generator/README.md

```diff
@@ -0,0 +1,76 @@
+# bot-template-generator
```

Comment:
Yeah thats the intention once we merge the initial version. It will likely only be used internally at first but having it publicly available will make it easier for internal devs to use it. I could see external devs using it as well down the line especially when we all custom feeds in the creation flow to populate the template selection screen and to get the open source community making third party templates post build. 

I was wondering the same thing as far as location, @ryanlengel what do you think, I believe we initially spoke about putting it in the tools directory but it is a generator so I could see the argument for both.

Reply:
Agree, since this will be published this should move to /generators. /tools should be reserved for internal scripts we use to maintain the repo/manage technical debt

Reply:
Okay I have moved it to the generators dir